tests/checksum/benchmark.py

```diff
@@ -48,7 +48,9 @@ def reset(self):
         '''
         print("start resetting benchmark...")
         with open(self.json_file, 'w') as outfile:
-            json.dump(self.data, outfile, sort_keys=True, indent=2)
+            # match output of manual reset
+            json.dump(self.data, outfile, indent=2)
+            outfile.write('\n')
```

Comment:
Could you please briefly explain this?

Reply:
When I used `reset_all_benchmarks.sh` I noticed that it did a lot of unnecessary line changes because it put the checksum in a different order. I updated this so the order will be the same as in the printout when a CI checksum test fails. I also added the newline so it looks normal in git.